metrique/README.md

```diff
@@ -4,9 +4,15 @@ metrique is a crate to emit unit-of-work metrics
 
 ## Getting Started (Applications)
 
-Most metrics your application records will be "unit of work" metrics. These are typically tied to the request/response scope. You declare a struct the represents the metrics you plan to capture over the course of the request and annotate it with `#[metrics]`
+Most metrics your application records will be "unit of work" metrics. These are typically tied to the request/response scope. You declare a struct the represents the metrics you plan to capture over the course of the request and annotate it with `#[metrics]`.
 
-This library exposes a wrapper `<MetricName>Guard` type (a type alias to [`AppendAndCloseOnDrop`]) that implicitly appends to a global queue when the struct is dropped. This wrapper in turn exposes other APIs to further tune behavior.
+The `#[metrics]` macro will make your struct implement the [`CloseEntry`] trait, which represents an entry that
+can be "closed" into an [`InflectableEntry`].
+
+The macro will also expose a wrapper `<MetricName>Guard` type (a type alias to [`AppendAndCloseOnDrop`]) that implicitly appends to a global queue when the struct is dropped. This wrapper in turn exposes other APIs to further tune behavior.
```

Comment:
this seems too low level for the getting started? I suspect many people will use library and never know what an `InflectableEntry` is. The key thing to capture in these docs is that instead of having some sort of global counters like you might expect from a metric library, you emit individual entries.